modules/programs/zellij.nix

```diff
@@ -0,0 +1,52 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.zellij;
+  yamlFormat = pkgs.formats.yaml { };
+in {
+  options = {
+    programs.zellij = {
+      enable = mkEnableOption "zellij";
+
+      package = mkOption {
+        type = types.package;
+        default = pkgs.zellij;
+        defaultText = literalExpression "pkgs.zellij";
+        example = literalExpression "pkgs.zellij";
+        description = ''
+          The zellij package to install.
+        '';
+      };
+
+      settings = mkOption {
+        type = yamlFormat.type;
+        default = { };
+        defaultText = "{}";
+        example = literalExpression ''
+          {
+            theme = "custom";
+            themes.custom.fg = 5;
+          }
+        '';
+        description = "The zellij configuration.";
```

Comment:
```suggestion
        description = ''
          Configuration written to
          <filename>$XDG_CONFIG_HOME/zellij/config.toml</filename>.
          </para><para>
          See <link xlink:href="https://www.zellij.sh/book/configuration.html" /> for the full list
          of options.
        '';
```

Reply:
Yup, the description is contained within a `<para>graph` and is automatically closed by one.
Adding new paragraphs is equivalent to adding a linebreak `<br>`.